### DISCUSSION TITLE
Either rename or add a disclaimer for the OpenAI extension

### DISCUSSION THREAD

**Discussion by Cohee1207** [oobabooga/text-generation-webui#4551] [General]
**Description**

Hi there, I see some confusion in the community related to the OpenAI API migration. People don't get that this is just an OpenAI-compatible API schema, instead, they think it'll be somehow routed via OpenAI's servers and/or moderated by them.

My proposals are the following:

1. Either rename the extension to remove the mentions of OpenAI and leave it just in the documentation that it is compatible with their client.
2. Or add a disclaimer on startup that it has no relation to OpenAI and all the prompts are local and won't be sent anywhere.

**Additional Context**

Some quotes from the community members.

> Also if the api is intended to be changed later on, are all of my chats sent to open ai? Because I would like to keep my chats local if possible because open AI is unbelievable with censorship. 

> Yesterday, I noticed a message that the current API is deprecated and will be replaced with OpenAI API on Nov 13th.
> Do I need to make OpenAI account? If so, can OpenAI see the information passing through?

**Comment by TechScorpion** [comment]
I think it could be renamed from an OpenAI one if it doesn't send stuff to them and it's not authorised by them. Though if there's going to be no other one might it limit it in some ways to make it always compatible with theirs?
